components/o-colors/src/scss/_use-cases.scss

```
@@ -21,6 +21,7 @@
 		('hero', ('background': 'wheat')),
 		('hero-opinion', ('background': 'oxford')),
 		('hero-highlight', ('background': 'claret')),
+		('ads', ('background': 'light-wheat')),
```

Instead, let's add a specific usecase for ads. We'll include a comment to explain how it's derived from the current palette. This makes it clear that, in affect, we're creating a colour that acts as wheat with opacity on an paper background. ([original use](https://github.com/Financial-Times/origami/tree/00b35280286192d08dcf564dbb21738bf8da02d8/components/o-colors#usecases))
```suggestion 
		('ads', ('background': #fcede0)), // oColorsMix(wheat, paper, 20);
```